### PR TITLE
Update README.

### PR DESCRIPTION
**Pull request by BigRoma**

### CODE REVIEW

File: README-ru.md [hunk 1]
```diff
@@ -3,14 +3,61 @@
...
+3) Нажмите кнопку **Fork**, которая расположена в правом верхнем углу, сделав таким образов ответвление проекта.
 
-* PR принимается только на превод одного языка
-* Для двух и более языков нужно сделать разные PR's
-* PR принимаются только для последней версии приложения
-* Для принятия PR нужно подтверждению минимум 3 ревьюверов
+## Редактирование файла перевода
+
+### Редактирование на GitHub
+1) Откройте нужный языковой файл нажав на него
```
Comment (AleksandrHorev) [inline README-ru.md]: add link https://github.com/tonlabs/TON-Surf-Localization/tree/development/src

File: README-ru.md [hunk 2]
```diff
@@ -3,14 +3,61 @@
...
+
+### Отправка перевода на компьютере
+1) Выполните первые два пункта в разделе **"Редактирование на GitHub"**
+2) Скопируйте содержимое файла из текcтового редактора внутрь окна редактирования.
+3) Нажмите кнопку **Commit changes**.
+
+## Создание запроса на изменение файла локализации
+### Правила создания 
+
+* PR принимается только на перевод одного языка.
```
Comment (AleksandrHorev) [inline README-ru.md]: don't use PR acronym, use only full version (Pull Request)

File: README-ru.md [hunk 3]
```diff
@@ -3,14 +3,61 @@
...
+* Для двух и более языков нужно сделать разные PR.
+* PR принимаются только для последней версии приложения.
+* Для принятия PR нужно подтверждению минимум 3 проверяющих.
+
+### Создание запроса на слияние (PR)
+1) Убедитесь что ваши изменения соответствуют формату JSON.
```
Comment (AleksandrHorev) [inline README-ru.md]: how can I verify it?
add link https://jsonformatter.curiousconcept.com/#

File: README-ru.md [hunk 4]
```diff
@@ -3,14 +3,61 @@
...
+2) Выберите вкладку Pull requests.
+3) Нажмите New pull request.
+4) Нажмите Create pull request. 
+5) Введите заголовок, опишите изменения.
+6) Нажмите кнопку Create pull request.
+
+
+**Внимание! Изменения которые вы вносите, появятся в приложении в течении 1-2 недель после принятия PR.**
```
Comment (AleksandrHorev) [inline README-ru.md]: в течении нескольких недель после одобрения и принятия PR.

**Review by AnnBoo — APPROVED** [review]

**Review by kkkhesha — APPROVED** [review]

**Review by AleksandrHorev — APPROVED** [review]

**Pull request merged**